script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -471,7 +471,7 @@ As I understand it, vampires that have been alive for a long time aren't very fu
 [sha:28abac3df6ca575115ad2774eb62e39807395543]{
 -- Page 546, Offset 17937.
 -- その手足を動かすごとに自分の力を消費していては話にならない。
- So it would go against that purpose if they needed to expend their own energy on those underlings.
+So it would go against that purpose if they needed to expend their own energy on those underlings.
```

Comment:
undo this. it will make the line render weird

i took it out of my pr

Reply:
huh ok. what a weird error